applications/openshift/api-server/api_server_auth_mode_rbac/rule.yml

```diff
@@ -0,0 +1,38 @@
+prodtype: ocp4
+
+title: Must have authorization-mode RBAC set
+
+description: 'Turn on Role Based Access Control'
```

Comment:
Description needs to be flushed out more. How is this done?

Reply:
Fixed locally, will mark as resolved when I push new patches.